src/openvic-simulation/dataloader/Dataloader.hpp

```diff
@@ -87,8 +90,12 @@ namespace OpenVic {
 		///
 		static fs::path search_for_game_path(fs::path hint_path = {});
 
-		/* In reverse-load order, so base defines first and final loaded mod last */
-		bool set_roots(path_vector_t const& new_roots);
+		/// @brief Sets the directories the dataloader should pull content from.
+		///
+		/// @param new_roots Dataloader roots in reverse-load order, so base defines first and final loaded mod last
+		/// @param new_replace_paths All base define paths that should be ignored entirely in favour of mods.
+		/// @return True if successful, false if failed.
+		bool set_roots(path_vector_t const& new_roots, path_vector_t const& new_replace_paths);
```

Comment:
Probably should become
```suggestion
		bool set_root(fs::path const& new_roots, path_vector_t const& new_replace_paths);
```
or something like that.